src/shared/logs-show.h

```diff
@@ -14,11 +14,11 @@
 #include "set.h"
 #include "time-util.h"
 
-typedef struct BootId {
```

Comment:
maybe add a comment here, that makes clear what kind of ids are stored in this, i.e. boot or invocation ids.